betelgeuse.py

```diff
@@ -41,6 +42,7 @@
 # Avoid SSL errors
 ssl._create_default_https_context = ssl._create_unverified_context
 
+INVALID_TEST_PLAN_CHARS_REGEX = re.compile(r'[\\/.:"<>|~!@#$?%^&\'*()+`,= ]')
```

Comment:
Ahh I see, have missed that space at the end.

In that case what about having the `INVALID_CHARS_REGEX` and for the test plan you also run `.replace(' ', '_')` after the re.sub?